models/gc_nnunet_pancreas/config/default.yml

```diff
@@ -0,0 +1,35 @@
+general:
+  version: 1.0
+  data_base_dir: /app/data
+  description: base configuration for GC NNUnet Pancreas model (dicom to dicom)
+
+execute:
+- DicomImporter
+- MhaConverter
+- GCNNUnetPancreasRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: '%Modality'
+
+  MhaConverter:
+    engine: panimg
+    targets: [dicom:mod=ct]
+
+  DsegConverter:
+    model_name: 'GC NNUnet Pancreas'
```

Comment:
I suggest to consistently omit quotes around strings whenever not required (which is only if the string starts with a particular character of meaning in yaml, e.g. `%`, `-`, `[`, ...) to enhance readability.,

Reply:
That's fine, I'll do that.